### PR TITLE
Cookie policy

### PR DESCRIPTION
**Pull request by twrdjpalmer**
Screenshot 2019-12-12 at 08 00 18

### CODE REVIEW

**Comment by KarlEST** [comment]
Can we see what the change looks like as well? 🙏

**Comment by twrdjpalmer** [comment]
@KarlEST updated

File: src/Footer/FooterBottom/FooterBottom.spec.js [hunk 1]
```diff
@@ -18,22 +18,18 @@ describe('Footer bottom', () => {
     );
   });
 
-  it('has items with logos', () => {
```
Comment (KarlEST) [inline src/Footer/FooterBottom/FooterBottom.spec.js]: Let's add a test that covers the moved TW logo part as well?

**Comment by KarlEST** [comment]
And on mobile they still stay under each other?

File: src/Footer/FooterBottom/FooterBottom.less [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+.footer-bottom-item {
+  width: 100%;
+}
+
+@media (min-width: @screen-sm-min) {
+  .footer-bottom-item {
+    width: 50%;
+    padding-right: 12px;
```
Comment (KarlEST) [inline src/Footer/FooterBottom/FooterBottom.less]: Is there some variable we can use here so we don't have to hardcode it?

**Pull request merged**